rfc6962-bis.xml

```diff
@@ -824,45 +825,52 @@ signatures.
-        SignatureType signature_type = tree_hash;
         LogID id;
         uint64 timestamp;
         uint64 tree_size;
         opaque root_hash[HASH_SIZE];
         SthExtensions extensions;
-    } TreeHeadSignature;</artwork>
-        </figure>
-        <t>
-          <spanx style="verb">version</spanx> is the version of the TreeHeadSignature structure. This version is v2. Note that <xref target="RFC6962">TreeHeadSignature v1</xref> did not include the <spanx style="verb">id</spanx> or <spanx style="verb">extensions</spanx> fields.
-        </t>
-        <t>
-          <spanx style="verb">timestamp</spanx> is the current time. The timestamp MUST be at least as recent as the most recent SCT timestamp in the tree. Each subsequent timestamp MUST be more recent than the timestamp of the previous update.
-        </t>
-        <t>
-          <spanx style="verb">tree_size</spanx> equals the number of entries in the new tree.
-        </t>
-        <t>
-          <spanx style="verb">root_hash</spanx> is the root of the Merkle Hash Tree.
-        </t>
-        <t>
-          <spanx style="verb">extensions</spanx> are future extensions to STH v2. Currently, no extensions are specified. If an implementation sees an extension that it does not understand, it SHOULD ignore that extension. Furthermore, an implementation MAY choose to ignore any extension(s) that it does understand.
-        </t>
-        <t>Each log MUST produce on demand a Signed Tree Head that is no older than the Maximum Merge Delay. However, Signed Tree Heads could be used to mark individual clients (by producing a new one for each query), so logs MUST NOT produce them more frequently than is declared in their metadata. In general, there is no need to produce a new Signed Tree Head unless there are new entries in the log, however, in the unlikely event that it receives no new submissions during an MMD period, the log SHALL sign the same Merkle Tree Hash with a fresh timestamp.
-        </t>
+        digitally-signed struct {
+            TreeHeadVersion version;
+            SignatureType signature_type = tree_hash;
+            LogID id;
+            uint64 timestamp;
+            uint64 tree_size;
+            opaque root_hash[HASH_SIZE];
+            SthExtensions extensions;
+        };
+    } SignedTreeHead;</artwork>
+         </figure>
+          <t>
+            <spanx style="verb">version</spanx> is the version of the SignedTreeHead structure. This version is v2. Note that <xref target="RFC6962">TreeHeadSignature v1</xref> only included the inner <spanx style="verb">digitally-signed struct</spanx> and did not include the <spanx style="verb">id</spanx> or <spanx style="verb">extensions</spanx> fields.
+          </t>
+          <t>
+            <spanx style="verb">timestamp</spanx> is the current time. The timestamp MUST be at least as recent as the most recent SCT timestamp in the tree. Each subsequent timestamp MUST be more recent than the timestamp of the previous update.
```

Comment:
FWIW, this "timestamp" definition is currently unchanged since 6962.  But yes, let's take this opportunity to make it more precise.  I'll copy the "...is the current NTP Time..." bits from the definition of the SCT timestamp.